scout/export/variant.py

```diff
@@ -1,26 +1,26 @@
+from pprint import pprint as pp
 
 
 def export_causatives(adapter, collaborator):
```

Comment:
Should we perhaps add the option to pass a list of collaborators to export causative variants from multiple institutes? Could do in separate PR

Reply:
Yes I also thought about that. One should also be able to export all variants from all institutes.